benches/flamegraph.rs

```diff
@@ -44,6 +43,8 @@ macro_rules! flamegraph_benchmarks {
 }
 
 flamegraph_benchmarks! {
-    dtrace: "tests/data/collapse-dtrace/results/dtrace-example.txt",
-    perf: "tests/data/collapse-perf/results/example-perf-stacks-collapsed.txt"
+    dtrace_reverse: ("tests/data/collapse-dtrace/results/dtrace-example.txt",
```

Comment:
I'd just call them `dtrace` and `perf`. Arguably we don't even need both of these, since it doesn't matter for the flamegraph benchmark what the source format was; only the folded stacks matter. I think we keep whichever one is larger. Thoughts?

Reply:
The perf one is quite a bit larger. Let's just keep that one.

Reply:
Do we want to call it something else instead of `perf` since there will only be one.

Reply:
Yeah, I think we call it just `flamegraph`?

Reply:
Since the group is also called `flamegraph` I thought it would appear in the benchmarks as `flamegraph/flamegraph`, but I just tried it and I was wrong.